contracts/contracts/auction/TokenDepositLocker.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.5.8;
+
+import "./DepositLocker.sol";
+import "../token/TrustlinesNetworkToken.sol";
+
+/*
+  The ETHDepositLocker contract locks ETH deposits
```

Comment:
The TokenDepositLocker contract locks Token deposits

Reply:
Changed